treeherder/perf/auto_perf_sheriffing/outcome_checker.py

```diff
@@ -39,9 +47,9 @@ def __init__(self):
     def check(self, record: BackfillRecord) -> OutcomeStatus:
         # TODO: get job_type from record when soft launch lands ---> job_type = record.job_type
         job_type = get_job_type(record)
-        from_time, to_time = self._get_push_timestamp_range(record.get_context())
+        from_time, to_time = Helper.get_push_timestamp_range(record.get_context())
```

Comment:
There's no need to refactor `OutcomeChecker`, as we already covered these exact methods in [PR 7070](https://github.com/mozilla/treeherder/pull/7070).